plugins/chat/app/jobs/regular/chat/channel_delete.rb

```diff
@@ -45,7 +45,7 @@ def delete_messages_and_related_records(chat_channel, chat_messages)
           # if the uploads are not used anywhere else they will be deleted
           # by the CleanUpUploads job in core
           ::DB.exec("DELETE FROM chat_uploads WHERE chat_message_id IN (#{message_ids.join(",")})")
-          ::UploadReference.where(target_id: message_ids, target_type: "Chat::Message").delete_all
+          ::UploadReference.where(target_id: message_ids, target_type: "ChatMessage").delete_all
```

Comment:
Should we use `Chat::Message.sti_name` to avoid 'magic strings' like this?

Reply:
Yes Im kinda hesitating to do so... although Im afraid Im misunderstand part of the logic somewhere and it backfires :D That's why I went with "not changing stuff" ™️

Reply:
Also I would have to change raw sql too